src/services/auth.ts

```diff
@@ -30,7 +28,7 @@ interface Response {
 
 type Callback = () => void;
```

Comment:
This can be substituted with NextFunction type, as well as the Response interface

Reply:
@Micosilent , done!